spring-data-jdbc-core/src/test/java/org/springframework/data/jdbc/core/OneToManyResultSetExtractorTest.java

```diff
@@ -94,12 +94,7 @@ protected Integer mapPrimaryKey(ResultSet rs) throws SQLException {
 
 		@Override
 		protected Integer mapForeignKey(ResultSet rs) throws SQLException {
-			if (rs.getObject("address.customer_id") == null) {
```

Comment:
Better if the user of this class don't have to implement the null-check like this.